packages/config/src/layer2s/arbitrum.ts

```diff
@@ -114,11 +115,17 @@ export const arbitrum: Layer2 = {
     },
     dataAvailability: RISK_VIEW.DATA_ON_CHAIN,
     upgradeability: RISK_VIEW.UPGRADABLE_ARBITRUM,
-    sequencerFailure: RISK_VIEW.SEQUENCER_TRANSACT_L1,
+    sequencerFailure: {
+      value: 'Transact using L1',
+      description: VALUES.ARBITRUM.getSequencerFailureString(),
+      references: [
+        'https://etherscan.io/address/0xD03bFe2CE83632F4E618a97299cc91B1335BB2d9#code#F1#L125',
```

Comment:
Nice one, we should have those references everywhere. Just a small nit-pick, I'm not sure if it would look better to link to a line of code rather than a comment.